UC-Chess/Chess.cs

```diff
@@ -47,16 +47,186 @@ public int getPos(int x, int y)
         public bool tryMove(int x, int y, int newX, int newY)
         {
             //If there is a piece to move
-            if(board[x,y] != 0)
+            if (board[x, y] != 0) //y==x x==y
             {
-                //Successfully move piece
-                board[newX, newY] = board[x, y];
-                board[x, y] = 0;
-                return true;
+                //to-do classes for each piece
+                switch (board[x, y])
```

Comment:
Some unreachable lines in this switch could be cleaned up, since both "return true" and "break" end the case